<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Vec3Widget</class>
 <widget class="QWidget" name="Vec3Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="labelTitle">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="lineEditX"/>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditY"/>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditZ"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
